flink-contrib/flink-storm/src/main/java/org/apache/flink/storm/wrappers/BoltWrapper.java

```diff
@@ -75,11 +78,13 @@
 	 * 
 	 * @param bolt
 	 *            The Storm {@link IRichBolt bolt} to be used.
+	 * @param componentId
+	 * @param streamId
 	 * @throws IllegalArgumentException
 	 *             If the number of declared output attributes is not with range [0;25].
 	 */
-	public BoltWrapper(final IRichBolt bolt) throws IllegalArgumentException {
-		this(bolt, null, (Collection<String>) null);
+	public BoltWrapper(final IRichBolt bolt, String componentId, String streamId) throws IllegalArgumentException {
+		this(bolt, componentId, streamId, null, (Collection<String>) null);
```

Comment:
This makes the interface quite hard to use and should not be necessary. The `TopologyContext` created in `open()` contains this information.